boot/makebootable.go

```diff
@@ -280,21 +280,42 @@ func makeBootable20RunMode(model *asserts.Model, rootdir string, bootWith *Boota
 	blVars := map[string]string{
 		"kernel_status": "",
 	}
-	if err := bl.SetBootVars(blVars); err != nil {
-		return fmt.Errorf("cannot set run system environment: %v", err)
-	}
 
 	ebl, ok := bl.(bootloader.ExtractedRunKernelImageBootloader)
```

Comment:
looking a the code I see a bunch of mentions about efi and symlinks before this point in comments, they need adjusting

Reply:
Updated

Reply:
I also tweaked some comments here to be more applicable and useful